code/_helpers/game.dm

```diff
@@ -16,11 +16,6 @@
 					return TRUE
 	return FALSE
 
-/proc/get_area(O)
-	RETURN_TYPE(/area)
-	var/turf/loc = get_turf(O)
-	return loc?.loc
```

Comment:
The new version clearly misses the subtle historical reference here to the superior implementation `O.loc:loc`. We are forgetting our heritage.